extensions/raycast-wallpaper/src/utils/raycast-wallpaper-utils.ts

```diff
@@ -0,0 +1,18 @@
+export type RaycastWallpaper = { title: string; url: string };
+
+export const raycastWallpaper = [
```

Comment:
To prevent you from having to update this extension every time we make a change to our wallpapers, I added a `.json` url to raycast.com: www.raycast.com/wallpapers.json. Can you maybe make the Extension work with this JSON endpoint?